### PR TITLE
Translate '2017-09-14 version realeases' (ko)

### PR DESCRIPTION
**Pull request by wagurano**
The pull requests involve translation of several posts, including 2.4.2, 2.3.5, 2.2.8 and vulnerability posts around these three versions.
1. 2017-09-14-json-heap-exposure-cve-2017-14064.md
1. 2017-09-14-openssl-asn1-buffer-underrun-cve-2017-14033.md
1. 2017-09-14-ruby-2-2-8-released.md
1. 2017-09-14-ruby-2-3-5-released.md
1. 2017-09-14-ruby-2-4-2-released.md
1. 2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md
1. 2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md

### CODE REVIEW

**Comment by yous** [comment]
@ruby/www-ruby-lang-org-i18n-ko

File: ko/news/_posts/2017-09-14-ruby-2-3-5-released.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+세부 내용은 아래 주제를 확인해주세요.
+
+* [CVE-2017-0898: Kernel.sprintf에서 버퍼 언더런 취약점 발생](/ko/news/2017/09/14/sprintf-buffer-underrun-cve-2017-0898/)
+* [CVE-2017-10784: WEBrick 베이직 인증의 이스케이프 시퀀스 삽입 취약점 발생](/ko/news/2017/09/14/webrick-basic-auth-escape-sequence-injection-cve-2017-10784/)
+* [CVE-2017-14033: OpenSSL ASN1 디코드할 때 버퍼 언더런 취약점 발생](/ko/news/2017/09/14/openssl-asn1-buffer-underrun-cve-2017-14033/)
+* [CVE-2017-14064: JSON을 생성할 때 힙 메모리를 노출하는 취약점](/ko/news/2017/09/14/json-heap-exposure-cve-2017-14064/)
+* [RubyGems의 취약점 다수 발견](/ko/news/2017/08/29/multiple-vulnerabilities-in-rubygems/)
+* libyaml 버전 0.1.7로 업데이트
+
+[체인지로그](http://svn.ruby-lang.org/repos/ruby/tags/v2_3_5/ChangeLog)를 확인해주세요.
```
Comment (yous) [inline ko/news/_posts/2017-09-14-ruby-2-3-5-released.md]: How about '변경기록'? '변경기록' or as-is 'ChangeLog' have been used.

File: ko/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+layout: news_post
+title: "CVE-2017-0898: Kernel.sprintf에서 버퍼 언더런 취약점 발생"
+author: "usa"
+translator: "wagurano"
+date: 2017-09-14 12:00:00 +0000
+tags: security
+lang: ko
+---
+
+`Kernel` 모듈의 `sprintf` 메소드에서 버퍼 언더런 취약점이 있습니다.
```
Comment (yous) [inline ko/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md]: 메소드에서 → 메소드에

File: ko/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+이 취약점은 CVE 아이디 [CVE-2017-0898](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-0898)로 할당하였습니다.
+
+## 세부 내용
+
+`*` 포함한 포맷 문자열을 고의로 조작하고 매우 큰 음수를 넘길 경우, 버퍼 언더런이 발생할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md]: How about '\`*\` 문자를 포함한 악의적인 포맷 문자열과 함께 매우 큰 음수를 넘길 경우'?

File: ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+layout: news_post
+title: "CVE-2017-10784: WEBrick 베이직 인증의 이스케이프 시퀀스 삽입 취약점 발생"
+author: "usa"
+translator: "wagurano"
+date: 2017-09-14 12:00:00 +0000
+tags: security
+lang: ko
+---
+
+루비 부가라이브러리 WEBrick의 베이직 인증에서 이스케이프 시퀀스 삽입 취약점이 있습니다.
```
Comment (yous) [inline ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: 베이직 인증에서 → 베이직 인증에

File: ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## 해당 버전
+
+* 루비 2.2 버전대: 2.2.7 이하
+* 루비 2.3 버전대: 2.3.4 이하
+* 루비 2.4 버전대: 2.4.1 이하
+* 리비전 58453 이전의 트렁크
+
+## 도움을 준 사람
+
+이 사안을 보고한 엔도 유스케 <mame@ruby-lang.org>에게 감사합니다.
```
Comment (yous) [inline ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: 엔도 유스케(\<mame@ruby-lang.org\>)에게

File: ko/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+## 해당 버전
+
+* 루비 2.2 버전대: 2.2.7 이하
+* 루비 2.3 버전대: 2.3.4 이하
+* 루비 2.4 버전대: 2.4.1 이하
+* 리비전 58453 이전의 트렁크
+
+## 도움을 준 사람
+
+이 사안을 보고한 [에얄 이트킨 aerodudrizzt](https://hackerone.com/aerodudrizzt)에게 감사합니다.
```
Comment (yous) [inline ko/news/_posts/2017-09-14-sprintf-buffer-underrun-cve-2017-0898.md]: 에얄 이트킨(aerodudrizzt)에게

File: ko/news/_posts/2017-09-14-ruby-2-3-5-released.md (hunk 1 above)
Comment (wagurano) [reply ko/news/_posts/2017-09-14-ruby-2-3-5-released.md]: '변경기록'이 좀 어색하여 'ChangeLog' 그대로가 좋아보입니다.

Comment (yous) [reply ko/news/_posts/2017-09-14-ruby-2-3-5-released.md]: Okay.

File: ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md (hunk 4 above)
Comment (wagurano) [reply ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: - English version: "Escape sequence injection vulnerability in the Basic authentication of WEBrick"
- Japanese version: "WEBrick の BASIC 認証におけるエスケープシーケンス挿入の脆弱性について"

Can I translate "in" and "に" as "에" to Korean? I will translate another statements.

Comment (yous) [reply ko/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md]: Yes. It's similar to '코드에 버그가 있습니다', while '코드에서 버그가 있습니다' is unnatural.

**Comment by hsbt** [comment]
Thanks a lot!

**Pull request merged**